.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+<!--
+Thanks for your contribution!
+
+Please ensure that any applicable requirements below are satisfied before submitting this pull request. This will help ensure a quick and efficient review cycle.
+-->
+
+**Description**
+
+_What type of change have you made?_
+
+- BREAKING:
+  - _Append BREAKING: to your PR title_
```

Comment:
I don't love the idea of asking people to prefix PR titles with the category :thinking:.

These labels are not mutually exclusive, for starters. If you've deprecated an API for security reasons, it doesn't seem useful to choose one and omit the other. They don't exactly look nice as titles either.

Reply:
Since the PR titles are used to create the changelog summary, I wanted to help standardize the way some of these items were reported. 

Would it make sense to keep the categories in the description but add BREAKING to the PR title if there are breaking changes? 

Something like:

```
**PR Title**
- A brief description of changes. If the PR has breaking changes add `BREAKING:` 
to the start of the PR title.

**Description**
- Change description and category list
```